components/profile/wallet-balance.tsx

```diff
@@ -0,0 +1,150 @@
+import { useEffect, useState } from "react";
+
+import { useTranslation } from "next-i18next";
+
+import BeProBlue from "assets/icons/bepro-blue";
+import OracleIcon from "assets/icons/oracle-icon";
+
+import { useAuthentication } from "contexts/authentication";
+import { useDAO } from "contexts/dao";
+import { useNetwork } from "contexts/network";
+
+import { formatNumberToCurrency } from "helpers/formatNumber";
+
+import { BEPRO_TOKEN, TokenInfo } from "interfaces/token";
+
+import { getCoinInfoByContract } from "services/coingecko";
+
+type TokenBalance = Partial<TokenInfo>;
+
+export default function WalletBalance() {
+  const { t } = useTranslation(["common", "profile"]);
+
+  const [tokens, setTokens] = useState<TokenBalance[]>([]);
+  const [totalEuro, setTotalEuro] = useState(0);
+  const [hasNoConvertedToken, setHasNoConvertedToken] = useState(false);
+
+  const { activeNetwork } = useNetwork();
+  const { wallet } = useAuthentication();
+  const { service: DAOService } = useDAO();
+
+  const FlexRow = ({ children, className = "" }) => <div className={`d-flex flex-row ${className}`}>{children}</div>;
```

Comment:
Would be interesting to have this (and Column) components in a component that could be used by everything else :)

Reply:
Yes, it is. I already changed it in the branch I'm working now.